ci/docker/builder/Dockerfile

```diff
@@ -5,7 +5,6 @@ LABEL org.opencontainers.image.source=https://github.com/NordSecurity/nordvpn-li
 RUN dpkg --add-architecture i386 && \
     dpkg --add-architecture arm64 && \
     dpkg --add-architecture armhf && \
-    dpkg --add-architecture armel && \
```

Comment:
Do we really drop `armel` arch support? If yes need to cleanup all other places e.g. `ci/archs.sh`  etc.

Reply:
It is not dropped. Necessary `armel` dependencies are installed with the workaround mentioned in line 22